### PR TITLE
ini children order callout

### PR DESCRIPTION
**Pull request by parkervcp**
If you don't get the order correct for child:parent then it will parse as yml and fail.

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: docs/docsite/rst/intro_inventory.rst [hunk 1]
```diff
@@ -179,6 +179,10 @@ Groups of Groups, and Group Variables
 It is also possible to make groups of groups using the ``:children`` suffix in INI or the ``children:`` entry in YAML.
 You can apply variables using ``:vars`` or ``vars:``:
 
+The order that the groups are in is important for the ini file format.
+You need to specify the child group then the parent group.
+The example below shows atlanta and raleigh then the southeast:children group they are in.
+If you specify the southeast:children then atlanta and raleigh it will fail parsing.
```
Comment (bcoca) [inline docs/docsite/rst/intro_inventory.rst:185]: this was a bug at one point, but should already be fixed

**Review by bcoca — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/intro_inventory.rst (hunk 1 above)
Comment (parkervcp) [reply docs/docsite/rst/intro_inventory.rst:185]: When I opened this it was still an issue. That is why I made it a callout.

Comment (bcoca) [reply docs/docsite/rst/intro_inventory.rst:185]: understood, but then its a bug report, which others did, so we fixed it.

we do not normally change the docs to fit the bug

Comment (parkervcp) [reply docs/docsite/rst/intro_inventory.rst:185]: I thought that it was intended. Once I discovered the issue it made sense that the individuals had to exist before you could group them.

Either way good to hear,

**Pull request closed without merging**